msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2023-05-09 09:37-0500\n"
"PO-Revision-Date: 2023-05-09 14:49\n"
"Last-Translator: \n"
"Language-Team: French\n"
"Language: fr_FR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: fr\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../bin/duplicity:106
msgid "INT intercepted...exiting."
msgstr "INT intercepté... sortie"

#: ../bin/duplicity:114
#, python-format
msgid "GPG error detail: %s"
msgstr "Détails de l’erreur GPG : %s"

#: ../bin/duplicity:124
#, python-format
msgid "User error detail: %s"
msgstr "Détails de l’erreur utilisateur : %s"

#: ../bin/duplicity:134
#, python-format
msgid "Backend error detail: %s"
msgstr "Détails de l’erreur de sauvegarde : %s"

#: ../bin/rdiffdir:64 ../duplicity/commandline.py:239
#, python-format
msgid "Error opening file %s"
msgstr "Erreur lors de l’ouverture du fichier %s"

#: ../bin/rdiffdir:131
#, python-format
msgid "File %s already exists, will not overwrite."
msgstr "Le fichier %s existe déjà, il ne sera pas écrasé."

#: ../duplicity/tempdir.py:160
#, python-format
msgid "Using temporary directory %s"
msgstr "Utilisation du dossier temporaire %s"

#: ../duplicity/tempdir.py:204
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Enregistrement (mktemp) du fichier temporaire %s"

#: ../duplicity/tempdir.py:226
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Enregistrement (mkstemp) du fichier temporaire %s"

#: ../duplicity/tempdir.py:258
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Oubli du fichier temporaire %s"

#: ../duplicity/tempdir.py:261
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Tente d'oublier le fichier temporaire inconnu %s - ceci est probablement un bogue."

#: ../duplicity/tempdir.py:280
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Suppression du fichier temporaire toujours en mémoire %s"

#: ../duplicity/tempdir.py:283
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Le nettoyage du fichier temporaire %s a échoué"

#: ../duplicity/tempdir.py:288
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr "Le nettoyage du dossier temporaire %s a échoué - ceci est probablement un bogue."

#: ../duplicity/backend.py:102
#, python-format
msgid "Import of %s %s"
msgstr "Importation de %s %s"

#: ../duplicity/backend.py:211
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Impossible d'initialiser le moteur : %s"

#: ../duplicity/backend.py:403
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Trace inverse de l'erreur précédente : %s"

#: ../duplicity/backend.py:419
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Abandonner après %s tentatives. %s: %s"

#: ../duplicity/backend.py:423
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backend.py:517
#, python-format
msgid "Reading results of '%s'"
msgstr "Lecture des résultats de « %s »"

#: ../duplicity/backend.py:543
#, python-format
msgid "Writing %s"
msgstr "Écriture de %s"

#: ../duplicity/backend.py:584
#, python-format
msgid "File %s not found locally after get from backend"
msgstr ""

#: ../duplicity/manifest.py:99
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Erreur fatale : l'hôte de la source de sauvegarde a changé.\n"
"Nom d'hôte courant : %s\n"
"Nom d'hôte précédent : %s"

#: ../duplicity/manifest.py:106
#, python-format
msgid "Fatal Error: Backup source directory has changed.\n"
"Current directory: %s\n"
"Previous directory: %s"
msgstr "Erreur fatale : le dossier de la source de sauvegarde a changé.\n"
"Dossier courant : %s\n"
"Dossier précédent : %s"

#: ../duplicity/manifest.py:116
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "Annulation parce que vous avez peut-être accidentellement essayé de sauvegarder deux jeux de données différents au même emplacement distant ou utilisé le même dossier d'archives. Si ceci est une erreur, utilisez l'option --allow-source-mismatch pour éviter de voir ce message."

#: ../duplicity/manifest.py:218
#, python-format
msgid "Found manifest volume %s"
msgstr ""

#: ../duplicity/manifest.py:225
#, python-format
msgid "Found %s volumes in manifest"
msgstr ""

#: ../duplicity/manifest.py:243
#, python-format
msgid "Manifest file '%s' is corrupt: File count says %d, File list contains %d"
msgstr ""

#: ../duplicity/manifest.py:262
msgid "Manifests not equal because different volume numbers"
msgstr "Fichiers manifest non égaux car les numéros de volume diffèrent"

#: ../duplicity/manifest.py:267
msgid "Manifests not equal because volume lists differ"
msgstr "Fichiers manifest non égaux car les listes de volume diffèrent"

#: ../duplicity/manifest.py:272
msgid "Manifests not equal because hosts or directories differ"
msgstr "Fichiers manifest non égaux car les hôtes ou les répertoires diffèrent"

#: ../duplicity/manifest.py:429
msgid "Warning, found extra Volume identifier"
msgstr "Attention, un identifiant de volume supplémentaire a été trouvé"

#: ../duplicity/manifest.py:455
msgid "Other is not VolumeInfo"
msgstr "Autre n'est pas VolumeInfo"

#: ../duplicity/manifest.py:458
msgid "Volume numbers don't match"
msgstr "Les numéros de volume ne correspondent pas"

#: ../duplicity/manifest.py:461
msgid "start_indicies don't match"
msgstr "les indices_de_départ ne correspondent pas"

#: ../duplicity/manifest.py:464
msgid "end_index don't match"
msgstr "les indices_de_fin ne correspondent pas"

#: ../duplicity/manifest.py:471
msgid "Hashes don't match"
msgstr "Les empreintes ne correspondent pas"

#: ../duplicity/util.py:166 ../duplicity/dup_main.py:847
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "ERREUR_IGNORÉE : attention : ignore l'erreur comme demandé : %s : %s"

#: ../duplicity/util.py:233
#, python-format
msgid "Releasing lockfile %s"
msgstr "Libération du fichier de verrouillage %s"

#: ../duplicity/backends/pydrivebackend.py:181
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "Arrière-plan PyDrive : plusieurs fichiers nommés « %s »."

#: ../duplicity/backends/multibackend.py:96
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend : Impossible d'analyser la chaîne de requêtes %s : %s "

#: ../duplicity/backends/multibackend.py:105
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultiBackend : Chaîne de requêtes %s non valide : plus d'une valeur pour %s"

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:160
#: ../duplicity/backends/multibackend.py:165
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:176
#, python-format
msgid "MultiBackend: Url %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:180
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MultiBackend : Impossible de charger le fichier de configuration %s : %s "

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MultiBackend : utiliser le magasin de sauvegarde %s"

#: ../duplicity/backends/multibackend.py:194
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:207
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:248
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr ""

#: ../duplicity/backends/multibackend.py:261
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MultiBackend : échec de l'écriture dans le magasin de sauvegarde #%s (%s), essayez #%s, exception : %s"

#: ../duplicity/backends/multibackend.py:268
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr ""

#: ../duplicity/backends/multibackend.py:275
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend : échec de l'écriture de %s. Essais faits dans tous les magasins de sauvegarde, aucun n'a fonctionné"

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MutiBackend : impossible de mettre %s sur %s à partir de %s"

#: ../duplicity/backends/multibackend.py:297
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MutiBackend : impossible d'obtenir %s. Essais faits dans tous les magasins de sauvegarde, aucun n'a fonctionné"

#: ../duplicity/backends/multibackend.py:307
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr ""

#: ../duplicity/backends/multibackend.py:310
#, python-format
msgid "Exception during list of %s: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:317
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MultiBackend : liste combinée : %s"

#: ../duplicity/backends/multibackend.py:346
#: ../duplicity/backends/multibackend.py:375
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend : la supression de %s a échoué. Essais faits dans tous les magasins de sauvegarde, aucun n'a réussi"

#: ../duplicity/backends/giobackend.py:109
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Connexion échouée, veuillez vérifier votre mot de passe : %s"

#: ../duplicity/backends/onedrivebackend.py:92
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:85
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:106
msgid "Option '--ssl-cacert-path' is not supported with python 2.7.8 and below."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:110
#, python-format
msgid "For certificate verification with python 2.7.8 or earlier a cacert database\n"
"file is needed in one of these locations: %s\n"
"Hints:\n"
"Consult the man page, chapter 'SSL Certificate Verification'.\n"
"Consider using the options --ssl-cacert-file, --ssl-no-check-certificate ."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:168
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:170
#, python-format
msgid "Using WebDAV directory %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:201
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:212
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:231
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:232
#: ../duplicity/backends/webdavbackend.py:255
#, python-format
msgid "WebDAV data length: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:235
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:241
#, python-format
msgid "WebDAV redirect to: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:243
msgid "WebDAV redirected 10 times. Giving up."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:248
msgid "WebDAV missing location header in redirect response."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:253
msgid "WebDAV retry request with authentification headers."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:254
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:258
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:275
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:279
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:377
#, python-format
msgid "Creating missing directory %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:394
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:439
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:461
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:481
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr ""

#: ../duplicity/diffdir.py:115 ../duplicity/diffdir.py:418
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Erreur %s pour l'obtention du delta pour %s"

#: ../duplicity/diffdir.py:130
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Obtention du delta de %s et %s"

#: ../duplicity/diffdir.py:181
#, python-format
msgid "A %s"
msgstr ""

#: ../duplicity/diffdir.py:188
#, python-format
msgid "M %s"
msgstr ""

#: ../duplicity/diffdir.py:210
#, python-format
msgid "Comparing %s and %s"
msgstr "Comparaison de %s et %s"

#: ../duplicity/diffdir.py:218
#, python-format
msgid "D %s"
msgstr ""

#: ../duplicity/commandline.py:78
#, python-format
msgid "Warning: Option %s is pending deprecation and will be removed in version 2.0.\n"
"Use of default filenames is strongly suggested."
msgstr ""

#: ../duplicity/commandline.py:84
#, python-format
msgid "Warning: Option %s is pending deprecation and will be removed in version 2.0.\n"
"--include-filelist and --exclude-filelist now accept globbing characters and should be used instead."
msgstr ""

#: ../duplicity/commandline.py:94
#, python-format
msgid "Warning: Option %s is pending deprecation and will be removed in version 2.0.\n"
"On many GNU/Linux systems, stdin is represented by /dev/stdin and\n"
"--include-filelist=/dev/stdin or --exclude-filelist=/dev/stdin could\n"
"be used as a substitute."
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/commandline.py:261 ../duplicity/commandline.py:271
#: ../duplicity/commandline.py:292 ../duplicity/commandline.py:388
#: ../duplicity/commandline.py:410 ../duplicity/commandline.py:421
#: ../duplicity/commandline.py:669 ../duplicity/commandline.py:717
#: ../duplicity/commandline.py:953
msgid "path"
msgstr "chemin"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a hidden GnuPG key. Example:
#. --hidden-encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/commandline.py:287 ../duplicity/commandline.py:294
#: ../duplicity/commandline.py:416 ../duplicity/commandline.py:650
#: ../duplicity/commandline.py:926
msgid "gpg-key-id"
msgstr "identifiant_clé_gpg"

#. Used in usage help to represent a pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <pattern>
#: ../duplicity/commandline.py:302 ../duplicity/commandline.py:444
#: ../duplicity/commandline.py:959
msgid "pattern"
msgstr ""

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/commandline.py:308 ../duplicity/commandline.py:317
#: ../duplicity/commandline.py:324 ../duplicity/commandline.py:392
#: ../duplicity/commandline.py:446 ../duplicity/commandline.py:453
#: ../duplicity/commandline.py:466 ../duplicity/commandline.py:922
msgid "filename"
msgstr "nom_de_fichier"

#. Used in usage help to represent a regular expression (regexp).
#: ../duplicity/commandline.py:331 ../duplicity/commandline.py:457
msgid "regular_expression"
msgstr "expression_régulière"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/commandline.py:335 ../duplicity/commandline.py:404
#: ../duplicity/commandline.py:544 ../duplicity/commandline.py:1008
msgid "time"
msgstr "temps"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:412 ../duplicity/commandline.py:521
#: ../duplicity/commandline.py:547 ../duplicity/commandline.py:658
#: ../duplicity/commandline.py:941
msgid "options"
msgstr ""

#: ../duplicity/commandline.py:430
#, python-format
msgid "Running in 'ignore errors' mode due to %s; please re-consider if this was not intended"
msgstr "Exécution en mode « ignorer les erreurs » à cause de %s ; veuillez reconsidérer si ceci n’était pas souhaité"

#. Used in usage help to represent an imap mailbox
#: ../duplicity/commandline.py:442
msgid "imap_mailbox"
msgstr "boîte_aux_lettres_IMAP"

#: ../duplicity/commandline.py:460
msgid "file_descriptor"
msgstr "descripteur_fichier"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/commandline.py:475 ../duplicity/commandline.py:497
#: ../duplicity/commandline.py:518 ../duplicity/commandline.py:524
#: ../duplicity/commandline.py:530 ../duplicity/commandline.py:583
#: ../duplicity/commandline.py:588 ../duplicity/commandline.py:592
#: ../duplicity/commandline.py:615 ../duplicity/commandline.py:621
#: ../duplicity/commandline.py:625 ../duplicity/commandline.py:699
#: ../duplicity/commandline.py:712 ../duplicity/commandline.py:725
#: ../duplicity/commandline.py:936
msgid "number"
msgstr "nombre"

#. Used in usage help (noun)
#: ../duplicity/commandline.py:478
msgid "backup name"
msgstr "nom_sauvegarde"

#: ../duplicity/commandline.py:610
msgid "policy"
msgstr ""

#: ../duplicity/commandline.py:628
msgid "Hot|Cool|Archive"
msgstr ""

#. noun
#: ../duplicity/commandline.py:631 ../duplicity/commandline.py:634
#: ../duplicity/commandline.py:907
msgid "command"
msgstr "commande"

#: ../duplicity/commandline.py:637
msgid "pyrax|cloudfiles"
msgstr ""

#: ../duplicity/commandline.py:661
msgid "pem formatted bundle of certificate authorities"
msgstr "Convertir au format PEM l'ensemble des autorités de certification"

#: ../duplicity/commandline.py:662
msgid "path to a folder with certificate authority files"
msgstr ""

#: ../duplicity/commandline.py:666
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr ""

#. Used in usage help. Example:
#. --timeout <seconds>
#. Used in usage help. Example:
#. --backend-retry-delay <seconds>
#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/commandline.py:674 ../duplicity/commandline.py:730
#: ../duplicity/commandline.py:976
msgid "seconds"
msgstr "secondes"

#. abbreviation for "character" (noun)
#: ../duplicity/commandline.py:680 ../duplicity/commandline.py:904
msgid "char"
msgstr "caract."

#: ../duplicity/commandline.py:736
#, python-format
msgid "Warning: Option %s is dangerous in that it allows corrupted manifests to\n"
"exist on the remote without being caught.  This could lead to problems\n"
"such as the inability to recover the backup in the future."
msgstr ""

#: ../duplicity/commandline.py:870
#, python-format
msgid "Using archive dir: %s"
msgstr "Utilisation du répertoire d’archive : %s"

#: ../duplicity/commandline.py:871
#, python-format
msgid "Using backup name: %s"
msgstr "Utilisation du nom de sauvegarde : %s"

#: ../duplicity/commandline.py:878
#, python-format
msgid "Command line error: %s"
msgstr "Erreur de ligne de commande : %s"

#: ../duplicity/commandline.py:879
msgid "Enter 'duplicity --help' for help screen."
msgstr "Entrez « duplicity --help » pour afficher l’écran d’aide."

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/commandline.py:892
msgid "absolute_path"
msgstr "chemin_absolu"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/commandline.py:896
msgid "alias"
msgstr ""

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:901
msgid "bucket_name"
msgstr "nom_de_bucket"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/commandline.py:912
msgid "container_name"
msgstr "nom_de_conteneur"

#. noun
#: ../duplicity/commandline.py:915
msgid "count"
msgstr "compte"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/commandline.py:918
msgid "directory"
msgstr "dossier"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/commandline.py:931
msgid "module"
msgstr ""

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:945
msgid "other.host"
msgstr "autre.hote"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:949
msgid "password"
msgstr "mot_de_passe"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:963
msgid "port"
msgstr ""

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:968
msgid "prefix"
msgstr "préfixe"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/commandline.py:972
msgid "relative_path"
msgstr "chemin_relatif"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/commandline.py:981
msgid "some_dir"
msgstr "dossier_quelconque"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:987
msgid "source_dir"
msgstr "dossier_source"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:992
msgid "source_url"
msgstr "url_source"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:998
msgid "target_dir"
msgstr "dossier_cible"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:1003
msgid "target_url"
msgstr "url_cible"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:1013
msgid "user"
msgstr "utilisateur"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/commandline.py:1016
msgid "account_id"
msgstr "identifiant_de_compte"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/commandline.py:1020
msgid "application_key"
msgstr "clef_d_application"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/commandline.py:1024 ../duplicity/dup_collections.py:521
msgid "remote"
msgstr "distant"

#. Header in usage help
#: ../duplicity/commandline.py:1043
msgid "Backends and their URL formats:"
msgstr "Moteurs et leurs formats URL :"

#. Header in usage help
#: ../duplicity/commandline.py:1081
msgid "Commands:"
msgstr "Commandes :"

#: ../duplicity/commandline.py:1108
#, python-format
msgid "Specified archive directory '%s' does not exist, or is not a directory"
msgstr "Le dossier d'archivage spécifié « %s » n'existe pas ou n'est pas un dossier"

#: ../duplicity/commandline.py:1117
#, python-format
msgid "Sign key should be an 8, 16 alt. 40 character hex string, like 'AA0E73D2'.\n"
"Received '%s' instead."
msgstr "La clef de signature devrait être une chaîne hexadécimale de 8, 16 ou 40 caractères, comme « AA0E73D2 ».\n"
"Au lieu de cela, la chaîne reçue était « %s »."

#: ../duplicity/commandline.py:1177
#, python-format
msgid "Restore destination directory %s already exists.\n"
"Will not overwrite."
msgstr "Le dossier de restauration cible %s existe déjà.\n"
"Il ne sera pas écrasé."

#: ../duplicity/commandline.py:1182
#, python-format
msgid "Verify directory %s does not exist"
msgstr "Vérifier que le dossier %s n'existe pas"

#: ../duplicity/commandline.py:1188
#, python-format
msgid "Backup source directory %s does not exist."
msgstr "Le dossier source de sauvegarde %s n'existe pas."

#: ../duplicity/commandline.py:1219
#, python-format
msgid "Command line warning: %s"
msgstr "Avertissement de la ligne de commande : %s"

#: ../duplicity/commandline.py:1219
msgid "Selection options --exclude/--include\n"
"currently work only when backing up,not restoring."
msgstr "L'option de sélection --exlude/--include\n"
"ne fonctionne actuellement que lors d'une sauvegarde et non pas lors d'une restauration."

#: ../duplicity/commandline.py:1255
#, python-format
msgid "GPG binary is %s, version %s"
msgstr "Le binaire GPG est %s, version %s"

#: ../duplicity/commandline.py:1282
#, python-format
msgid "Bad URL '%s'.\n"
"Examples of URL strings are \"scp://user@host.net:1234/path\" and\n"
"\"file:///usr/local\".  See the man page for more information."
msgstr "Mauvaise URL « %s ».\n"
"Un exemple d'URL est : « scp://utilisateur@hote.net:1234/chemin » et\n"
"« file:///usr/local ». Veuillez voir la page de manuel pour plus d'informations."

#: ../duplicity/commandline.py:1312
msgid "Main action: "
msgstr "Action principale : "

#: ../duplicity/librsync.py:196
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr ""

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Mauvais chaîne d'intervalle « %s »\n\n"
"Les intervalles sont spécifiés comme 2Y (2 ans) ou 2h30min (2,5 heures). Les\n"
"caractères spéciaux autorisés sont s, m, h, D, W, M et Y. Voir la page de manuel\n"
"pour plus d'informations."

#: ../duplicity/dup_time.py:69
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Mauvais chaîne de temps « %s »\n\n"
"Les chaînes de temps acceptées sont les intervalles (comme « 3D64s »), des chaînes\n"
"au format w3-datetime, comme « 2002-04-26T04:22:01-07:00 » (les chaînes comme\n"
"« 2002-04-26T04:22:01 » sont aussi acceptées - duplicity utilisera le \n"
"fuseau horaire courant), ou des dates ordinaires comme 2/4/1997 ou 2001-04-23\n"
"(plusieurs combinaisons sont possibles, mais le mois précède toujours le jour)."

#: ../duplicity/dup_main.py:120
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Réutiliser la PASSPHRASE configurée comme SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:127
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Réutiliser la SIGN_PASSPHRASE configurée comme PASSPHRASE"

#: ../duplicity/dup_main.py:168
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE non établie, demande à l'utilisateur."

#: ../duplicity/dup_main.py:183
msgid "GnuPG passphrase for signing key:"
msgstr "Phrase de passe GnuPG pour la clé de signature :"

#: ../duplicity/dup_main.py:188
msgid "GnuPG passphrase for decryption:"
msgstr ""

#: ../duplicity/dup_main.py:193
msgid "Retype passphrase for signing key to confirm: "
msgstr "Veuillez retaper la phrase de passe pour la clé de signature pour confirmer : "

#: ../duplicity/dup_main.py:195
msgid "Retype passphrase for decryption to confirm: "
msgstr ""

#: ../duplicity/dup_main.py:198
msgid "First and second passphrases do not match!  Please try again."
msgstr "La première et la seconde phrases de passe ne correspondent pas ! Veuillez réessayer."

#: ../duplicity/dup_main.py:205
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "Impossible d'utiliser une phrase de passe vide avec un chiffrement symétrique ! Veuillez réessayer."

#: ../duplicity/dup_main.py:263
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "Le fichier %s est terminé dans le jeu de sauvegarde.\n"
"Le redémarrage continue sur fichier %s."

#: ../duplicity/dup_main.py:272
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "Le fichier %s est manquant dans le jeu de sauvegarde.\n"
"Le redémarrage continue sur le fichier %s."

#: ../duplicity/dup_main.py:326
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d, retrying."
msgstr ""

#: ../duplicity/dup_main.py:331
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Le fichier % s a été endommagé pendant le téléversement."

#: ../duplicity/dup_main.py:371
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "Redémarrage de sauvegarde, mais les paramètres de chiffrement actuels ne correspondent pas aux paramètres originaux"

#: ../duplicity/dup_main.py:394
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr ""

#: ../duplicity/dup_main.py:397
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Redémarrage après le volume %s, le fichier %s, le bloc %s"

#: ../duplicity/dup_main.py:475
#, python-format
msgid "Processed volume %d"
msgstr "Volume %d traité"

#: ../duplicity/dup_main.py:625
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "Erreur fatale : impossible de démarrer la sauvegarde incrémentale. Les anciennes signatures sont introuvables pour la méthode incrémentale indiquée"

#: ../duplicity/dup_main.py:629
msgid "No signatures found, switching to full backup."
msgstr "Aucune signature de sauvegarde trouvée, bascule vers une sauvegarde complète."

#: ../duplicity/dup_main.py:643
msgid "Backup Statistics"
msgstr "Statistiques de sauvegarde"

#: ../duplicity/dup_main.py:751
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s introuvable dans l'archive - aucun fichier n'a été restauré."

#: ../duplicity/dup_main.py:755
msgid "No files found in archive - nothing restored."
msgstr "Aucun fichier trouvé dans l’archive – rien n'a été restauré."

#: ../duplicity/dup_main.py:792
#, python-format
msgid "Processed volume %d of %d"
msgstr "Le volume %d sur %d a été traité"

#: ../duplicity/dup_main.py:837
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Données non valides - empreinte %s non vérifiée pour le fichier :"

#: ../duplicity/dup_main.py:840
#, python-format
msgid "Calculated hash: %s"
msgstr "Empreinte calculée : %s"

#: ../duplicity/dup_main.py:841
#, python-format
msgid "Manifest hash: %s"
msgstr "Empreinte du fichier manifest : %s"

#: ../duplicity/dup_main.py:892
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Le volume a été signé par la clé %s et non pas %s"

#: ../duplicity/dup_main.py:924
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Vérification complète : %s, %s."

#: ../duplicity/dup_main.py:925
#, python-format
msgid "%d file(s) compared"
msgstr ""

#: ../duplicity/dup_main.py:926
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:944
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Aucun fichier étranger détecté, aucune suppression effectuée lors du nettoyage."

#: ../duplicity/dup_main.py:949
msgid "Deleting these file(s) from backend:"
msgstr ""

#: ../duplicity/dup_main.py:958
msgid "Found the following file(s) to delete:"
msgstr ""

#: ../duplicity/dup_main.py:959
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Exécutez duplicity à nouveau avec l’option --force pour supprimer réellement."

#: ../duplicity/dup_main.py:1002
msgid "There are backup set(s) at time(s):"
msgstr "Il y a un ou plusieurs jeux de sauvegarde datés du :"

#: ../duplicity/dup_main.py:1004
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Qui ne peuvent être supprimés car de nouveaux jeux en dépendent."

#: ../duplicity/dup_main.py:1008
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "La chaîne de sauvegarde active actuelle est plus ancienne que la date indiquée. Cependant elle ne sera pas supprimée. Pour ôter toutes vos sauvegardes, purgez manuellement leur dépôt."

#: ../duplicity/dup_main.py:1021
msgid "No old backup sets found, nothing deleted."
msgstr "Aucun ancien jeu de sauvegarde n’a été trouvé, rien n’a été supprimé."

#: ../duplicity/dup_main.py:1024
msgid "Deleting backup chain(s) at time:"
msgstr ""

#: ../duplicity/dup_main.py:1034
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:1036
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:1039
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Suppression de la chaîne de signature complète %s"

#: ../duplicity/dup_main.py:1041
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Suppression de la chaîne de sauvegarde complète %s"

#: ../duplicity/dup_main.py:1047
msgid "Found old backup chain(s) at the following time:"
msgstr ""

#: ../duplicity/dup_main.py:1049
msgid "Rerun command with --force option to actually delete."
msgstr "Veuillez relancer la commande avec l'option --force pour forcer la suppression."

#: ../duplicity/dup_main.py:1072
msgid "No old backup sets found."
msgstr ""

#: ../duplicity/dup_main.py:1087
#, python-format
msgid "Signature %s already replicated"
msgstr ""

#: ../duplicity/dup_main.py:1094 ../duplicity/dup_main.py:1133
#, python-format
msgid "Replicating %s."
msgstr ""

#: ../duplicity/dup_main.py:1121
#, python-format
msgid "Backupset %s already replicated"
msgstr ""

#: ../duplicity/dup_main.py:1269
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Supprime le %s local (ne fait pas autorité au niveau du serveur)."

#: ../duplicity/dup_main.py:1274
#, python-format
msgid "Unable to delete %s: %s"
msgstr "Impossible de supprimer %s : %s"

#: ../duplicity/dup_main.py:1308 ../duplicity/dup_temp.py:271
#, python-format
msgid "Failed to read %s: %s"
msgstr "Impossible de lire %s : %s"

#: ../duplicity/dup_main.py:1321
#, python-format
msgid "Copying %s to local cache."
msgstr "Copie de %s vers le cache local."

#: ../duplicity/dup_main.py:1369
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Les métadonnées locales et distantes sont déjà synchronisées. Aucune synchronisation nécessaire."

#: ../duplicity/dup_main.py:1374
msgid "Synchronizing remote metadata to local cache..."
msgstr "Synchronisation des métadonnées distantes vers le cache local..."

#: ../duplicity/dup_main.py:1387
msgid "Sync would copy the following from remote to local:"
msgstr "La synchronisation va copier localement les éléments suivants à partir de l’hôte distant :"

#: ../duplicity/dup_main.py:1390
msgid "Sync would remove the following spurious local files:"
msgstr "La synchronisation va supprimer les fichiers parasites suivants :"

#: ../duplicity/dup_main.py:1434
msgid "Unable to get free space on temp."
msgstr "Impossible de déterminer l’espace libre dans le dossier temporaire."

#: ../duplicity/dup_main.py:1442
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr "Il y a %d d’espace temporaire disponible, la sauvegarde nécessite environ %d."

#: ../duplicity/dup_main.py:1445
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr "Il y a %d d’espace temporaire disponible, la sauvegarde va nécessiter environ %d."

#: ../duplicity/dup_main.py:1453
msgid "Unable to get max open files."
msgstr "Impossible d’obtenir le maximum de fichiers ouverts."

#: ../duplicity/dup_main.py:1457
#, python-format
msgid "Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr "Le nombre maximum de fichiers ouverts pour %s est trop bas, il devrait être >= 1024.\n"
"Utilisez « ulimit -n 1024 » (ou supérieur) pour corriger.\n"

#: ../duplicity/dup_main.py:1509
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr "REDÉMARRAGE : échec de téléchargement du premier volume avant le délai de fin.\n"
"              Le redémarrage est impossible... lancement de la sauvegarde depuis le début."

#: ../duplicity/dup_main.py:1515
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "REDÉMARRAGE : échec de téléchargement de %d volumes sur %d avant le délai de fin.\n"
"              Redémarrage de la sauvegarde au volume %d."

#: ../duplicity/dup_main.py:1522
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "REDÉMARRAGE : état de sauvegarde impossible :\n"
"              le fichier manifest indique %d volumes, le distant en a %d.\n"
"              Redémarrage impossible... duplicity va nettoyer la dernière\n"
"              sauvegarde partielle puis reprendre la sauvegarde depuis le début."

#: ../duplicity/dup_main.py:1543
msgid "\n"
"PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr "\n"
"PYTHONOPTIMIZE dans l'environnement empêche Duplicity de\n"
"reconnaître ses propres sauvegardes. Supprimez PYTHONOPTIMIZE\n"
"de l'environnement et relancez la sauvegarde.\n"

#: ../duplicity/dup_main.py:1566
#, python-format
msgid "Acquiring lockfile %s"
msgstr ""

#: ../duplicity/dup_main.py:1629
#, python-format
msgid "Last %s backup left a partial set, restarting."
msgstr "La dernière sauvegarde %s a laissé un jeu partiel, redémarrage."

#: ../duplicity/dup_main.py:1633
#, python-format
msgid "Cleaning up previous partial %s backup set, restarting."
msgstr "Nettoyage du dernier jeu partiel de sauvegarde %s, redémarrage."

#: ../duplicity/dup_main.py:1645
msgid "Last full backup date:"
msgstr "Date de la dernière sauvegarde complète :"

#: ../duplicity/dup_main.py:1647
msgid "Last full backup date: none"
msgstr "Date de la dernière sauvegarde complète : aucune"

#: ../duplicity/dup_main.py:1650
msgid "Last full backup is too old, forcing full backup"
msgstr "La dernière sauvegarde complète est trop ancienne, forçage d’une nouvelle sauvegarde complète"

#: ../duplicity/dup_main.py:1699
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "Lors de l'utilisation du chiffrement symétrique, le mot de passe de signature doit correspondre au mot de passe de chiffrement."

#: ../duplicity/asyncscheduler.py:70
#, python-format
msgid "instantiating at concurrency %d"
msgstr "instanciation au niveau de concurrence %d"

#: ../duplicity/asyncscheduler.py:97
msgid "inserting barrier"
msgstr "insertion d’une barrière"

#: ../duplicity/asyncscheduler.py:146
msgid "running task synchronously (asynchronicity disabled)"
msgstr "exécution synchronisée de la tâche (asynchronisme désactivé)"

#: ../duplicity/asyncscheduler.py:152
msgid "scheduling task for asynchronous execution"
msgstr "planification d’une tâche exécutée de façon asynchrone"

#: ../duplicity/asyncscheduler.py:181
msgid "task completed successfully"
msgstr "tâche achevée avec succès"

#: ../duplicity/asyncscheduler.py:192
msgid "a previously scheduled task has failed; propagating the result immediately"
msgstr "une tâche planifiée précédemment a échoué ; propagation immédiate du résultat"

#: ../duplicity/asyncscheduler.py:215 ../duplicity/asyncscheduler.py:236
#, python-format
msgid "active workers = %d"
msgstr "exécutants actifs = %d"

#: ../duplicity/asyncscheduler.py:256
#, python-format
msgid "task execution done (success: %s)"
msgstr "exécution de tâche réalisée (succès : %s)"

#: ../duplicity/gpginterface.py:241
msgid "Threading not available -- zombie processes may appear"
msgstr "Mise en fil d'exécution non disponible -- des processus zombies pourraient apparaître"

#: ../duplicity/gpginterface.py:710
#, python-format
msgid "GPG process %d terminated before wait()"
msgstr "Le processus GPG %d s'est arrêté avant wait()"

#: ../duplicity/robust.py:64
#, python-format
msgid "Error listing directory %s"
msgstr "Erreur de listage du dossier %s"

#: ../duplicity/dup_collections.py:186 ../duplicity/dup_collections.py:200
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "BackupSet.delete : %s manquant"

#: ../duplicity/dup_collections.py:225
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Erreur fatale : aucun fichier manifest présent pour la sauvegarde la plus récente"

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "Erreur fatale : le fichier manifest distant ne correspond pas à celui en local. Soit le jeu de sauvegarde distant, soit le dossier local d’archive a été corrompu."

#: ../duplicity/dup_collections.py:240 ../duplicity/dup_collections.py:270
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr ""

#: ../duplicity/dup_collections.py:247
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Erreur fatale : ni le fichier manifest local, ni le distant ne sont lisibles."

#: ../duplicity/dup_collections.py:258
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Traitement du manifeste local %s (%s)"

#: ../duplicity/dup_collections.py:272
#, python-format
msgid "Processing remote manifest %s (%s)"
msgstr "Traitement du manifeste distant %s(%s)"

#: ../duplicity/dup_collections.py:368
msgid "Preferring Backupset over previous one!"
msgstr "Jeu de sauvegarde préféré au précédent !"

#: ../duplicity/dup_collections.py:371
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Ignore le jeu de sauvegarde incrémentale (date_début : %s ; nécessaire : %s)"

#: ../duplicity/dup_collections.py:376
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Jeu de sauvegarde incrémentale ajouté (date_début : %s ; date_fin : %s)"

#: ../duplicity/dup_collections.py:446
msgid "Chain start time: "
msgstr "Date de début de chaîne : "

#: ../duplicity/dup_collections.py:447
msgid "Chain end time: "
msgstr "Date de fin de chaîne : "

#: ../duplicity/dup_collections.py:448
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Nombre de jeux de sauvegarde contenus : %d"

#: ../duplicity/dup_collections.py:450
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Nombre total de volumes contenus : %d"

#: ../duplicity/dup_collections.py:452 ../duplicity/dup_collections.py:1228
msgid "Type of backup set:"
msgstr "Type de jeu de sauvegarde :"

#: ../duplicity/dup_collections.py:452 ../duplicity/dup_collections.py:1228
msgid "Time:"
msgstr "Date :"

#: ../duplicity/dup_collections.py:452
msgid "Num volumes:"
msgstr "Nombre de volumes :"

#: ../duplicity/dup_collections.py:456 ../duplicity/dup_collections.py:1234
msgid "Full"
msgstr "Complète"

#: ../duplicity/dup_collections.py:459 ../duplicity/dup_collections.py:1236
msgid "Incremental"
msgstr "Incrémentale"

#: ../duplicity/dup_collections.py:519
msgid "local"
msgstr ""

#: ../duplicity/dup_collections.py:676
msgid "Collection Status"
msgstr "État de la collection"

#: ../duplicity/dup_collections.py:678
#, python-format
msgid "Connecting with backend: %s"
msgstr "Connexion au serveur central : %s"

#: ../duplicity/dup_collections.py:680
#, python-format
msgid "Archive dir: %s"
msgstr "Dossier d’archive : %s"

#: ../duplicity/dup_collections.py:682
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr ""

#: ../duplicity/dup_collections.py:685
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Chaîne secondaire %d sur %d :"

#: ../duplicity/dup_collections.py:691
msgid "Found primary backup chain with matching signature chain:"
msgstr "Chaîne primaire de sauvegarde trouvée avec la signature de chaîne correspondante :"

#: ../duplicity/dup_collections.py:695
msgid "No backup chains with active signatures found"
msgstr "Aucune chaîne de sauvegarde avec les signatures actives n’a été trouvée"

#: ../duplicity/dup_collections.py:698
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr ""

#: ../duplicity/dup_collections.py:700
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr ""

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:703
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Ceux-ci peuvent être supprimés en exécutant duplicity avec la commande « cleanup »."

#: ../duplicity/dup_collections.py:706
msgid "No orphaned or incomplete backup sets found."
msgstr "Aucun jeu orphelin ou incomplet de sauvegarde n’a été trouvé."

#: ../duplicity/dup_collections.py:722
#, python-format
msgid "%d file(s) exists on backend"
msgstr ""

#: ../duplicity/dup_collections.py:730
#, python-format
msgid "%d file(s) exists in cache"
msgstr ""

#: ../duplicity/dup_collections.py:781
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "Avertissement, rejet du dernier jeu de sauvegarde à cause de l’absence du fichier de signatures."

#: ../duplicity/dup_collections.py:804
msgid "Warning, found the following local orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:809
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:814
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Avertissement, signatures trouvées mais aucun fichier de sauvegarde correspondant"

#: ../duplicity/dup_collections.py:818
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "Avertissement, jeux de sauvegarde incomplets trouvés, probablement laissés par des sessions interrompues"

#: ../duplicity/dup_collections.py:822
msgid "Warning, found the following orphaned backup file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:835
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Extraction des chaînes de sauvegarde depuis la liste des fichiers : %s"

#: ../duplicity/dup_collections.py:847
#, python-format
msgid "File %s is part of known set"
msgstr "Le fichier %s fait partie d’un jeu connu"

#: ../duplicity/dup_collections.py:850
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Le fichier %s ne fait pas partie d’un jeu connu ; création d’un nouveau jeu"

#: ../duplicity/dup_collections.py:855
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Fichier ignoré (rejeté par le jeu de sauvegarde) « %s »"

#: ../duplicity/dup_collections.py:871
#, python-format
msgid "Found backup chain %s"
msgstr "Chaîne de sauvegarde %s trouvée"

#: ../duplicity/dup_collections.py:876
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Jeu %s ajouté à la chaîne préexistante %s"

#: ../duplicity/dup_collections.py:880
#, python-format
msgid "Found orphaned set %s"
msgstr "Jeu orphelin %s trouvé"

#: ../duplicity/dup_collections.py:1030
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr ""

#: ../duplicity/dup_collections.py:1226
#, python-format
msgid "File: %s"
msgstr "Fichier : %s"

#: ../duplicity/dup_collections.py:1227
#, python-format
msgid "Total number of backup: %d"
msgstr "Nombre total de sauvegardes : %d"

#: ../duplicity/dup_collections.py:1228 ../duplicity/dup_collections.py:1254
msgid "Type of file change:"
msgstr "Type de changement de fichier :"

#: ../duplicity/dup_collections.py:1252
#, python-format
msgid " Backup set time: %s"
msgstr ""

#: ../duplicity/dup_collections.py:1253
#, python-format
msgid "Total number of changes: %d"
msgstr ""

#: ../duplicity/dup_collections.py:1254
msgid "File:"
msgstr ""

#: ../duplicity/patchdir.py:83 ../duplicity/patchdir.py:88
#, python-format
msgid "Patching %s"
msgstr "En train de réparer %s"

#: ../duplicity/patchdir.py:558
#, python-format
msgid "Error '%s' patching %s"
msgstr "Erreur « %s » pendant la réparation de %s"

#: ../duplicity/patchdir.py:633
#, python-format
msgid "Writing %s of type %s"
msgstr "Écriture %s de type %s"

#: ../duplicity/selection.py:117
#, python-format
msgid "Skipping socket %s"
msgstr "Passer le socket %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "Erreur d'initialisation du fichier %s"

#: ../duplicity/selection.py:125 ../duplicity/selection.py:164
#: ../duplicity/selection.py:583
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Erreur d'accès au fichier %s éventuellement verrouillé"

#: ../duplicity/selection.py:179
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Attention : la base %s n'existe pas, continue"

#: ../duplicity/selection.py:182 ../duplicity/selection.py:203
#: ../duplicity/selection.py:206
#, python-format
msgid "Selecting %s"
msgstr "Sélection %s"

#: ../duplicity/selection.py:259
#, python-format
msgid "The last file selection option is the filter option %s, which will have no\n"
"effect as there are no subsequent file selection options. Exiting because\n"
"this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:268
#, python-format
msgid "Only these filter mode options were specified:\n"
"    %s\n"
"Case sensitive globbing is the default behaviour and so this has no effect.\n"
"Exiting because this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:342
#, python-format
msgid "Fatal Error: The file specification\n"
"    %s\n"
"cannot match any files in the base directory\n"
"    %s\n"
"Useful file specifications begin with the base directory or some\n"
"pattern (such as '**') which matches the base directory."
msgstr "Erreur fatale : La spécification du fichier\n"
"    %s\n"
"ne concorde avec aucun fichier du dossier de base\n"
"    %s\n"
"Les spécifications de fichier utiles commence par le dossier de base ou par un\n"
"motif (tel que « ** ») qui correspond avec le dossier de base."

#: ../duplicity/selection.py:351
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "Erreur fatale pendant le traitement de l'expression\n"
"%s"

#: ../duplicity/selection.py:368
#, python-format
msgid "Reading files to backup from list %s"
msgstr ""

#: ../duplicity/selection.py:390
#, python-format
msgid "Files-from list contains the absolute path:\n"
"    %s\n"
"All paths specified in a files-from list must be given relative to the backup\n"
"source path."
msgstr ""

#: ../duplicity/selection.py:399
msgid "Files-from list specified which contains no files, the backup will be empty as\n"
"a result. Exiting as this probably isn't what you meant,"
msgstr ""

#: ../duplicity/selection.py:409
#, python-format
msgid "Last selection expression:\n"
"    %s\n"
"only specifies that files be included.  Because the default is to\n"
"include all files, the expression is redundant.  Exiting because this\n"
"probably isn't what you meant."
msgstr "Dernière expression sélectionnée :\n"
"    %s\n"
"ne spécifie seulement que les fichiers inclus. Parce que l'option\n"
"par défaut inclut tous les fichiers, l'expression est redondante.\n"
"Quitte parce que ce n'est probablement pas ce que vous vouliez."

#: ../duplicity/selection.py:468
#, python-format
msgid "Reading %s filelist %s"
msgstr ""

#: ../duplicity/selection.py:509
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Erreur en compilant l'expression régulière %s"

#: ../duplicity/lazy.py:355
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Attention : ancienindice %s >= nouvelindice %s"

#: ../duplicity/lazy.py:430
#, python-format
msgid "Error '%s' processing %s"
msgstr "Erreur « %s » lors du traitement de %s"

#: ../duplicity/lazy.py:440
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Passe %s à cause d'une erreur précédente"

#: ../duplicity/path.py:115
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "Avertissement : %s nombres de développement non valides (0x%X), considérés comme étant (0, 0)."

#: ../duplicity/path.py:245 ../duplicity/path.py:306
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Attention : %s a un mtime négatif, il sera traité comme étant 0."

#: ../duplicity/path.py:370
msgid "Difference found:"
msgstr "Différence trouvée :"

#: ../duplicity/path.py:379
#, python-format
msgid "New file %s"
msgstr "Nouveau fichier %s"

#: ../duplicity/path.py:382
#, python-format
msgid "File %s is missing"
msgstr "Fichier %s manquant"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Le fichier %%s est de type %s, %s attendu"

#: ../duplicity/path.py:391 ../duplicity/path.py:417
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Le fichier %%s a les permissions %s, %s attendues"

#: ../duplicity/path.py:396
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Le fichier %%s a un mtime %s, %s attendu"

#: ../duplicity/path.py:404
#, python-format
msgid "Data for file %s is different"
msgstr "les données pour le fichier %s sont différentes"

#: ../duplicity/path.py:412
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Le lien symbolique %%s pointe vers %s, %s attendu"

#: ../duplicity/path.py:421
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Le fichier de périphérique %%s a le numéro %s, %s attendu"

#: ../duplicity/path.py:613
#, python-format
msgid "Making directory %s"
msgstr "Création du dossier %s"

#: ../duplicity/path.py:623
#, python-format
msgid "Deleting %s"
msgstr "Suppression de %s"

#: ../duplicity/path.py:632
#, python-format
msgid "Touching %s"
msgstr "Touche %s"

#: ../duplicity/path.py:639
#, python-format
msgid "Deleting tree %s"
msgstr "Suppression de l'arbre %s"

